acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/dsl/node/ProcessPantheonNodeRunner.java

```diff
@@ -92,6 +92,8 @@ public void startNode(final PantheonNode node) {
       params.add(node.getPrivacyParameters().getEnclavePublicKeyFile().getAbsolutePath());
       params.add("--privacy-precompiled-address");
       params.add(String.valueOf(node.getPrivacyParameters().getPrivacyAddress()));
+      params.add("--privacy-marker-transaction-signing-key-file");
```

Comment:
I prefer a flag `--random-marker-signing` (or similar) which is a boolean that defaults to `false`. If it is `false` then the marker transaction signing key defaults to `--node-private-key-file` i.e. uses the fixed transaction signing strategy.

Otherwise, use the random transaction signing strategy.

Reply:
That being said - this is also a viable approach. The downside is that in most cases a user needs to set `--node-private-key-file` and `--privacy-marker-transaction-signing-key-file` to the same value.

Reply:
The problem I see is when we go to mainnet, the account with funds to sign transactions may not be the same key as that used by the node.
Having said that - I suspect this needs to change to be "--signer-url", and Pantheon will need to submit the marker transaction to EthSigner to do the key-management stuff... Not sure how that plays with Acceptance Tests :/